lumeer-core/src/main/java/io/lumeer/core/facade/ProjectFacade.java

```diff
@@ -102,7 +102,7 @@ public Project getProject(final String projectCode) {
 
    public List<Project> getProjects() {
       User user = authenticatedUser.getCurrentUser();
-      DatabaseQuery query = DatabaseQuery.createBuilder(user.getUsername())
+      DatabaseQuery query = DatabaseQuery.createBuilder(user.getEmail())
```

Comment:
In the future, we would probably want to use user ID for this. But it is out of the scope of this PR.